File: PRAK104-2310817210019-NaufalElyzar.c
#include <stdio.h>

int main() {
    int HargaSepatuA = 400000;
    int HargaSepatuB = 350000;

    int DiskonSepatuA = 13;
    int DiskonSepatuB = 21;

    int HargaDiskonSepatuA = HargaSepatuA - (HargaSepatuA * DiskonSepatuA / 100);
    int HargaDiskonSepatuB = HargaSepatuB - (HargaSepatuB * DiskonSepatuB / 100);

    printf("Harga sepatu A adalah %d\n", HargaSepatuA);
    printf("Harga sepatu B adalah %d\n", HargaSepatuB);
    printf("Sepatu A mendapat diskon %d%% sehingga harganya menjadi %d\n", DiskonSepatuA, HargaDiskonSepatuA);
    printf("Sepatu B mendapat diskon %d%% sehingga harganya menjadi %d\n", DiskonSepatuB, HargaDiskonSepatuB);

    return 0;
}

File: PRAK104-2310817210019-NaufalElyzar.py
HargaSepatuA = 400000
HargaSepatuB = 350000

DiskonA = 0.13  
DiskonB = 0.21  

HargaDiskonA = HargaSepatuA - (HargaSepatuA * DiskonA)
HargaDiskonB = HargaSepatuB - (HargaSepatuB * DiskonB)

print(f"Harga sepatu A adalah
      {HargaSepatuA}")
print(f"Harga sepatu B adalah {HargaSepatuB}")
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {HargaDiskonA:.0f}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {HargaDiskonB:.0f}")
